crates/router/src/types/transformers.rs

```diff
@@ -1068,6 +1068,18 @@ impl ForeignTryFrom<domain::MerchantConnectorAccount>
                 })
                 .transpose()?
                 .map(api_models::admin::AdditionalMerchantData::foreign_from),
+            connector_wallets_details: item
+                .connector_wallets_details
+                .map(|data| {
+                    let data = data.into_inner();
+                    serde_json::Value::parse_value::<api_models::admin::ConnectorWalletDetails>(
+                        data.expose(),
+                        "ConnectorWalletDetails",
+                    )
```

Comment:
Can't we call `data.expose().parse_value::<api_models::admin::ConnectorWalletDetails>("ConnectorWalletDetails")`?

Reply:
Can you make the same change elsewhere in the file?